deploy/ui/form.json

```diff
@@ -709,7 +709,7 @@
                     },
                     {
                       "label": "GC max metaspace size",
-                      "type": "integer",
+                      "type": "text",
```

Comment:
does this one also have to be changed?
https://github.com/kiegroup/kie-cloud-operator/blob/3ee4418812914035c88836c6682a52a737105233/deploy/ui/form.json#L1365-L1370

Reply:
Yes, missed this. Changed now :)